plugins/command_cedict.py

```diff
@@ -0,0 +1,120 @@
+
+        await message.reply(cedict_response.formatted_result,
+                            reply_markup=reply_markup, quote=True)
+    else:
+        await message.reply("Usage: \n<code>/cedict your query</code>"
+                            "\n\nFor example:\n<code>/cedict China</code>"
+                            "\n\nYou can also use <code>/dic</code> instead of"
+                            " <code>/cedict</code>",
+                            quote=True)
+
+
+@Client.on_callback_query(filters.regex("cedict_next_button_pressed")
+                          | filters.regex("cedict_previous_button_pressed"))
+async def callback_query_cedict_button_pressed(_,
+                                               callback_query: CallbackQuery):
+    current_page_first_entry_number = re.search(
+        '[0-9]+',
+        callback_query.message.text
+    ).group()
+
+    current_page = int(
+        int(current_page_first_entry_number) / CEDICT_ENTRIES_PER_PAGE
+    )
+
+    if callback_query.data == "cedict_next_button_pressed":
+        next_page = current_page + 1
+    else:
+        next_page = current_page - 1
+    original_query = " ".join(
+        callback_query.message.reply_to_message.text.split(" ")[1:])
+
+    cedict_response = get_cedict_result(original_query,
+                                        next_page,
+                                        CEDICT_ENTRIES_PER_PAGE)
+    has_next_page = cedict_response.has_next_page
+    has_previous_page = next_page > 0
+
+    if has_next_page and has_previous_page:
+        reply_markup = _make_previous_and_next_page_buttons()
+    elif has_previous_page:
+        reply_markup = _make_previous_page_button()
+    else:
+        reply_markup = _make_next_page_button()
+
+    await callback_query.message.edit_text(cedict_response.formatted_result,
+                                           reply_markup=reply_markup)
+
+
+def _make_next_page_button():
+    return InlineKeyboardMarkup(
+        [
+            [
+                InlineKeyboardButton(
+                    "Next page / 下一頁",
+                    callback_data="cedict_next_button_pressed"
+                )
+            ]
+        ]
+    )
+
+
+def _make_previous_page_button():
+    return InlineKeyboardMarkup(
+        [
+            [
+                InlineKeyboardButton(
+                    "Previous page / 上一頁",
+                    callback_data="cedict_previous_button_pressed"
+                )
+            ]
+        ]
+    )
+
+
+def _make_previous_and_next_page_buttons():
+    return InlineKeyboardMarkup(
+        [
+            [
+                InlineKeyboardButton(
+                    "Next page / 下一頁",
+                    callback_data="cedict_next_button_pressed"
+                )
+            ],
+            [
+                InlineKeyboardButton(
+                    "Previous page / 上一頁",
+                    callback_data="cedict_previous_button_pressed"
+                )
```

Comment:
instead of `Next page / 下一頁`, you can just use emoji.

https://emojipedia.org/next-track-button/

`pyrogram.emoji.NEXT_TRACK_BUTTON`

https://emojipedia.org/last-track-button/

`pyrogram.emoji.LAST_TRACK_BUTTON`

I would use

`f"{emoji.NEXT_TRACK_BUTTON} Next"`

unless it's intended to have both languages. anyway it's up to you. by using emoji will make it more intuitive, maybe just prepend the emoji.